pilot/pkg/bootstrap/server.go

```diff
@@ -1243,14 +1245,66 @@ func (s *Server) initWorkloadTrustBundle() {
 	})
 
 	// IstioCA: Explicitly add roots corresponding to CA
-	if s.CA != nil {
+	rootCertFile := path.Join(LocalCertDir.Get(), "root-cert.pem")
```

Comment:
is the root cert always in this folder+kept up to date? a bit surprised in all cases it is, but SGTM if so.

Reply:
On deeper inspection, we need not always have a root-cert. Will need to refactor this.